### DISCUSSION TITLE
clipboard manager

### DISCUSSION THREAD

**Discussion by RiteshChepuri** [i3/i3#5821] [Q&A] [answered]
Does i3 have any clipboard manager where we select all the previously copied texts?

**Comment by vishal340** [comment]
use greenclip maybe. I am new user myself but greenclip works super well. it has i3 specific settings in aur.

**Comment by RiteshChepuri** [comment]
seems like it requires rofi, I'm not considering using rofi. anyhow I'll I give it a try. Thank you for info.

**Comment by okraits** [comment, marked as answer]
i3 is a window manager. Its job is to manage windows. You can try something like parcellite.
